worker-document-interface/src/main/java/com/hpe/caf/worker/document/model/Field.java

```diff
@@ -99,6 +99,30 @@
      */
     boolean hasValues();
 
+    /**
+     * Sets the value in this field.<p>
+     * The value will be stored using the UTF-8 encoding.
```

Comment:
Let's make the difference between `add()` and `set()` clear in the JavaDocs - possibly even link to `add()` and vice versa.